lib/embulk/input/mixpanel.rb

```diff
@@ -66,7 +66,8 @@ def self.transaction(config, &control)
           retry_limit: config.param(:retry_limit, :integer, default: 5),
           latest_fetched_time: latest_fetched_time,
           incremental: incremental,
-          slice_range: config.param(:slice_range, :integer, default: 7)
+          slice_range: config.param(:slice_range, :integer, default: 7),
+          job_start_time: Time.now.to_i*1000
```

Comment:
Do you consider time zone difference?
I found that task has `timezone` option and wondering if `Time.now.to_i` works when `timezone` is set.

Reply:
Time.now.to_i will convert current time to epoch. Epoch is universal So I think it's ok without timezone

Reply:
OK. It should work in this case.

Reply:
@taikhuu `Time.now.to_i` is epoch, but `Time.now` still timezone-dependent, ie. depends on runtime timezone:
```>> Time.now
=> 2017-09-26 16:15:10 +0700
>> Time.now.utc
=> 2017-09-26 09:15:14 UTC
>> Time.now.to_i
=> 1506417319
>> Time.now.utc.to_i
=> 1506417322```

Reply:
Anyways, like @sakama said, it should work for this case, unless you're comparing to another datetime.

Reply:
The reason I raised this, because I remembered RuboCop complained about Time usage without explicit timezone, more details: http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/TimeZone